remove ABV isn't and others (#390)
carpentries-incubator/python-intermediate-development: _episodes/53-improvement-through-feedback.md

**Must have**,
**Should have**,
**Could have**,

> and **Will not have**.

Reviewer: ```suggestion
and **Won't have**.
```

- *Could Have* (CH) -
  these are desirable but not necessary,
  and each of these will be included in this timebox if it can be achieved.

> - *Will not Have* (WH) -

Reviewer: ```suggestion
- *Won't Have* (WH) -
```

### GitHub's Milestones

> Once we have decided on those we will work on (i.e. not Will not Haves),

Reviewer: ```suggestion
Once we have decided on those we will work on (i.e. not Won't Haves),
```

![Milestones in GitHub](../fig/github-assign-milestone.png){: .image-with-shadow width="1000px"}

Let us now use Milestones to plan and prioritise our team's next sprint.

> ## Exercise: Prioritise!
>
> Put your stakeholder hats on, and as a team apply MoSCoW to the repository issues
> to determine how you will prioritise effort to resolve them in the allotted time.
> Try to stick to the 60/20/20 rule,

> > and assign all issues you'll be working on (i.e. not `Will not Haves`) to a new milestone,

Reviewer: ```suggestion
> and assign all issues you will be working on (i.e. not `Won't Haves`) to a new milestone,
```

{: .challenge}

Depending on how many issues were registered on your repository,

> it is likely you Will not have resolved all the issues in this first milestone.

Reviewer: ```suggestion
it is likely you will not have resolved all the issues in this first milestone.
```
